Cargo.toml

```diff
@@ -60,18 +60,22 @@ webpki-roots = "0.18.0"
 version = "1.6.2"
 optional = true
 
+[dependencies.async-std-resolver]
```

Comment:
This is trust-dns's implementation that uses async-std instead of tokio. Since there were slight changes to tokio's I/O traits that made it a little tricky to implement for async-std's types manually, I figured we might as well just switch to using the built-in support.